testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -123,6 +145,18 @@ class MiniAppDisplayActivity : BaseActivity() {
         miniappPermissionCallback.invoke(isGranted)
     }
 
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+        if (requestCode == externalWebViewReqCode && resultCode == Activity.RESULT_OK) {
+            data?.let {intent ->
+                if (intent.hasExtra(WebViewActivity.loadUrlTag))
```

Comment:
Could we contain some of this functionality within the SDK so that we can assure the Mini App is receiving the correct data? We could change `emitResult` to accept the `Intent` instead like `emitResult(intent: Intent)`.